lib/try_hook.rb

```diff
@@ -70,4 +71,15 @@ def create_tempfile
     file
   end
 
+  def required_commands
+    %w(bash sh)
+  end
+
+  def default_allowed_commands
+    %w(whoami cat cp grep ln ls mkdir mv rm rmdir sed touch git head tail wc)
+  end
+
+  def allowed_commands
```

Comment:
I would leave allowed_commands as
`@custom_allowed_commands.presence || default_allowed_commands`

and 

¿available_commands? as `[allowed_commands, required_commands].join " "`

just for clarity

Reply:
sounds reasonable!